services/ui-src/src/components/reports/ReportPageFooter.tsx

```diff
@@ -86,24 +87,21 @@ interface Props {
 
 const sx = {
   footerBox: {
-    marginTop: "3.5rem",
-  },
-  buttonFlex: {
-    justifyContent: "space-between",
-    marginY: "1.5rem",
-  },
-  floatButtonRight: {
-    justifyContent: "right",
-    marginY: "1.5rem",
+    marginTop: "2.5rem",
+    paddingTop: "1.5rem",
   },
   arrowIcon: {
     width: "1rem",
   },
   button: {
     width: "8.25rem",
   },
+  nextButton: {
+    minWidth: "8.25rem",
+    marginLeft: "auto",
+  },
   praStatement: {
     fontSize: "0.875rem",
-    paddingTop: "2rem",
+    marginTop: "1rem",
```

Comment:
I noticed the gap between PRA and the continue button in Figma is huge! I know 1rem is what we have in MFP, so I'm not sure what to follow, just throwing it out there. I also think 1rem will look better when the previous button is gone.


![Screenshot 2024-11-19 at 9 25 57 AM](https://github.com/user-attachments/assets/a4f88685-eec7-43ca-9932-1c91adb5156b)